dll/win32/kernel32/client/proc.c

```diff
@@ -1489,13 +1564,31 @@ ExitProcess(IN UINT uExitCode)
     BASE_API_MESSAGE ApiMessage;
     PBASE_EXIT_PROCESS ExitProcessRequest = &ApiMessage.Data.ExitProcessRequest;
 
+    HMODULE winsock_module = NULL;
+    PPEB Peb = NtCurrentPeb();
+    typedef int (WINAPI *wsclcall)(VOID);
+    wsclcall DynWSACleanup;
+    
     ASSERT(!BaseRunningInServerProcess);
 
     _SEH2_TRY
     {
         /* Acquire the PEB lock */
         RtlAcquirePebLock();
 
+	    if (Peb->SpareUlong > 0) // inherited socket count
+        {
+    		DPRINT("ExitPocess: inherited sockets present, calling WSACleanup().\n");
```

Comment:
```suggestion
            DPRINT("ExitPocess: inherited sockets present, calling WSACleanup()\n");
```
Also you erroneously indented partly with tabs here instead of spaces. Please fix that everywhere where you did as well.